dynawo/sources/Modeler/Common/test/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ add_custom_target(${MODULE_NAME}-tests
     "DYNAWO_DICTIONARIES=dictionaries_mapping"
     "DYNAWO_ADEPT_INSTALL_DIR=${ADEPT_HOME}"
     "DYNAWO_INSTALL_OPENMODELICA=${INSTALL_OPENMODELICA}"
```

Comment:
Same question as the equivalent PR on private side: why not using only 1 environment variable (OPENMODELICAHOME)?

Reply:
Maybe we could. OPENMODELICAHOME is not necessary if you compiled OpenModelica on your computer but is needed to use the one from a distrib so we started to add it in a lot of places... I don't know the extend of the refactoring. env variables with the prefix DYNAWO was also a nice way to keep environment in control

Reply:
ok let's keep this topic for another PR